apis/crd/v1alpha1/cloudentityselector_webhook_test.go

```diff
@@ -0,0 +1,533 @@
+// Copyright 2022 Antrea Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	"context"
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+)
+
+var _ = Describe("CloudEntitySelectorWebhook", func() {
+	Context("Validation of CES configuration", func() {
+		var (
+			testAccountNamespacedName = types.NamespacedName{Namespace: "namespace01", Name: "account01"}
+			testSecretNamespacedName  = types.NamespacedName{Namespace: "namespace01", Name: "secret01"}
+			credentials               = "credentials"
+			testAbc                   = "abc"
+			testDef                   = "def"
+			testXyz                   = "xyz"
+			errorMsg01                = "vpc matchName with either vm matchID or vm matchName is not supported"
```

Comment:
I am suggesting to put err msg in file apis/crd/v1alpha1/cloudentityselector_webhook.go. This way any change in that file will automatically update the test

Reply:
Done